### PR TITLE
Add widget to show about.yml percentage adoption

### PR DESCRIPTION
**Pull request by monfresh**
Note: The CSS is a little messed up and will need to be adjusted by someone with better frontend skills than me.

The CSS for the "Last Updated" at info is in the `.updated-at` class in `nested_list.scss` I believe.
The CSS for the Meter widget is in `widgets/meter/meter.scss`. These are all provided by default by the Dashing framework.

I also updated the gridster and knob JS plugins to the latest versions.

Here is what it looks like:
screen shot 2015-09-22 at 6 45 00 pm

### CODE REVIEW

**Comment by monfresh** [comment]
cc @arowla @mbland @jessieay. This adds the percentage adoption widget Alison suggested. It's a WIP in that the front end has some visual issues. See my original comment. Perhaps one of you can fix while I'm gone.

File: .jshintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+assets/javascripts/gridster/jquery.gridster.min.js
+assets/javascripts/jquery.knob.min.js
```
Comment (jessieay) [inline .jshintignore:2]: should these be in `lib` since they are outside libraries?

File: jobs/repos_without_about_yml.rb [hunk 2]
```diff
@@ -17,4 +17,10 @@
   end
 
   send_event('repos_without_about_yml', unordered: true, items: items)
+
+  total_org_repos = ENV['GITHUB_ORG_REPOS'].split(',').size.to_f
+  repos_without_about = repos_without_about_yml.size.to_f
+  percentage_adoption = (1 - repos_without_about / total_org_repos) * 100
+
+  send_event('about_percentage', value: percentage_adoption.round(2))
```
Comment (jessieay) [inline jobs/repos_without_about_yml.rb:25]: test for this?

File: .jshintignore (hunk 1 above)
Comment (monfresh) [inline .jshintignore:2]: Good point, but I think the convention for outside libraries is to place them in `vendor/assets`, right?

File: jobs/repos_without_about_yml.rb (hunk 2 above)
Comment (monfresh) [inline jobs/repos_without_about_yml.rb:25]: I would if I could figure out how to. I haven't been able to find any examples in the wild of anyone testing what gets sent to a widget. I asked on the Dashing repo and never got a response: https://github.com/Shopify/dashing/issues/615

File: .jshintignore (hunk 1 above)
Comment (jessieay) [inline .jshintignore:2]: yes, `vendor` would make sense. I am not always clear on what belongs in each. this seems like a helpful link: http://guides.rubyonrails.org/asset_pipeline.html#asset-organization

File: jobs/repos_without_about_yml.rb (hunk 2 above)
Comment (jessieay) [inline jobs/repos_without_about_yml.rb:25]: wow! that is a mystery. Good on you for trying, at least.

File: .jshintignore (hunk 1 above)
Comment (monfresh) [inline .jshintignore:2]: I tried to put them in `vendor` but Sprockets did not like that. I couldn't get it to work easily, so I think we should make this a separate exercise in a separate PR.

**Comment by monfresh** [comment]
I fixed the styling issues and am merging in now.

**Comment by arowla** [comment]
:tada:
